sdk/nodejs/example/simple-entrypoint/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "tmp",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc",
+    "start": "node --loader ts-node/esm ./main.mts"
```

Comment:
This is totally fine, but I'm curious if you think every `start` script users write will be exactly this? If so, could we just put `node --loader ts-node/esm ./main.mts` in the typescript runtime on [this line](https://github.com/dagger/dagger/pull/5305/files#diff-53e3c4d2665984237716f1e4be37edc950485b8cf2c89d9e89230ced7f342cfdR24)?

Reply:
It could be this or something else, by using the script `start`, we let the user choose how to start the ci.
This way, he may rename his file `ci.ts` or `ci.mts` or use a different loader or anything. What we actually need is to launch the script, we can let the user define its own way.